src/SFA.DAS.FAA.Domain/Apply/DisabilityConfident/GetDisabilityConfidentApiRequest.cs

```diff
@@ -2,14 +2,15 @@
 
 namespace SFA.DAS.FAA.Domain.Apply.DisabilityConfident
 {
-    public class GetDisabilityConfidentApiRequest(Guid ApplicationId, Guid CandidateId) : IGetApiRequest
+    public class GetDisabilityConfidentApiRequest(Guid applicationId, Guid candidateId) : IGetApiRequest
     {
-        public string GetUrl => $"applications/{ApplicationId}/disability-confident?candidateId={CandidateId}";
+        public string GetUrl => $"applications/{applicationId}/disability-confident?candidateId={candidateId}";
     }
 
     public class GetDisabilityConfidentApiResponse
     {
         public bool? ApplyUnderDisabilityConfidentScheme { get; set; }
         public string EmployerName { get; set; } = null!;
+        public bool? IsSectionCompleted { get; set; }
```

Comment:
Although the outer api endpoint may include this, that is an error. We shouldn't add this here, but instead this page should have its own request and response.

Reply:
IsSectionCompleted is the property helps to retain/preserve the radio button values irrespective of the user come from edit page or task list page, 

so i believe this essential.

Reply:
ok, i will create a new Query handler solely for this confirmation page to get the status and other details, i believe that would satisfy the "Separation of concerns"